### DISCUSSION TITLE
Initial page at connection

### DISCUSSION THREAD

**Discussion by donsaimon** [BimmerGestalt/AAIdrive#304] [Ideas]
Is it possible to start the AAI app directly in the music playing display?
Now, at car start, on the car display it is displayed the page with all music sources available and If you want to see the music playing display you have to press "media" button near the iDrive controller.
It would be great if at car power on the display of music playing is automatically displayed (as it happens when USB key or radio FM are selected as music sources)

**Comment by hufman** [comment]
The app API is very constrained, and generally only allows apps to change windows in response to a user input. The one exception (popup event) is limited to special popup windows, and ID5 further restricted this to only be used by BMW Connected. This makes it difficult to implement this idea, but thank you for the suggestion!

**Reply by donsaimon** [reply]
Thank you Walter
